mlsshared.sty

```diff
@@ -143,7 +143,7 @@
         include,pragma,undef,warning}
 }[keywords,comments,strings,directives]
 
-% Note: \lstinline!...! used for inline Modelica using [short]modelica
+% Note: \lstinline!…! used for inline Modelica using [short]modelica
```

Comment:
This change seems just odd. It seems the new text uses some odd Unicode symbol for horizontal ellipsis, and I would prefer to keep text - especially comments as plain ASCII when possible.

Reply:
I removed the comment entirely, since the same thing is also expressed (in a better way) where the dialect is set as the default language.  Resolving.